src/components/transcript/index.tsx

```diff
@@ -220,23 +239,21 @@ const Transcript = ({ reviewData }: { reviewData: UserReviewData }) => {
           },
         }
       );
-      onSuccessCallback && onSuccessCallback();
+      onSuccessCallback?.();
```

Comment:
nit: same here. The verbose version is simple to read

Reply:
Same as https://github.com/bitcointranscripts/transcription-review-front-end/pull/288#discussion_r1594307159